swarm/swarm.go

```diff
@@ -213,9 +226,29 @@ func (self *Swarm) Start(srv *p2p.Server) error {
 		}
 	}
 
+	go self.metricsLoop()
+
+	startCounter.Inc(1)
 	return nil
 }
 
+// metricsLoop periodically sends metrics about storage
+func (self *Swarm) metricsLoop() {
+	ticker := time.NewTicker(metricsTimeout)
+
+	go func() {
+		for _ = range ticker.C {
+			self.sendMetrics()
+		}
+	}()
+}
+
+func (self *Swarm) sendMetrics() {
```

Comment:
This method name is misleading, it doesn't seem to be sending the metrics anywhere, just updating some metrics?

Reply:
Fixed.